docs/how-to/configure-text-generation-models.md

```diff
@@ -0,0 +1,160 @@
+# Configure Text Generation Models
+
+KubeAI supports the following engines for text generation models (LLMs, VLMs, ..):
+
+- vLLM (Recommended for GPU)
+- Ollama (Recommended for CPU)
+- Need something else? Please file an issue on [GitHub](https://github.com/substratusai/kubeai).
+
+There are 2 ways to install a text generation model in KubeAI:
+- Use Helm with the `kubeai/models` chart.
+- Use `kubectl apply -f model.yaml` to install a Model CRD.
+
+KubeAI comes with pre-validated and optimized Model configurations for popular text generation models. These models are available in the `kubeai/models` Helm chart and
+are also published as raw manifests in the `manifests/model` directory.
+
+You can also easily define your own models using the Model CRD directly or by using the `kubeai/models` Helm chart.
+
+## Install a Text Generation Model using Helm
+
+KubeAI provides a `kubeai/models` chart that contains the pre-configured models.
+
+You can take a look at all the pre-configured models in the chart's default values file.
+
+```bash
+helm show values kubeai/models
```

Comment:
They would need to have loaded the chart first right?

Reply:
That's a given if you're trying to configure models after you've installed KubeAI. Instlaling KubeAI requires adding the KubeAI helm repo.